{"version":1,"ops":[{"type":5,"author":{"id":"04135dc995e2fdcd322aaf4268dbf04630a3cc7c"},"timestamp":1460526376,"metadata":{"github-id":"MDEyOkxhYmVsZWRFdmVudDYyNTIxNzA2OQ=="},"added":["bug"],"removed":[]},{"type":5,"author":{"id":"04135dc995e2fdcd322aaf4268dbf04630a3cc7c"},"timestamp":1460526376,"metadata":{"github-id":"MDEyOkxhYmVsZWRFdmVudDYyNTIxNzA3MA=="},"added":["blocking"],"removed":[]},{"type":3,"author":{"id":"04135dc995e2fdcd322aaf4268dbf04630a3cc7c"},"timestamp":1460536284,"metadata":{"github-id":"MDEyOklzc3VlQ29tbWVudDIwOTMwMTg5Ng==","github-url":"https://github.com/perl11/cperl/issues/134#issuecomment-209301896"},"message":"Still fails on darwin DEBUGGING clang: https://travis-ci.org/perl11/cperl/jobs/122719501\n-\u003e exit code 11 of t/op/signatures.t","files":null},{"type":3,"author":{"id":"04135dc995e2fdcd322aaf4268dbf04630a3cc7c"},"timestamp":1460538567,"metadata":{"github-id":"MDEyOklzc3VlQ29tbWVudDIwOTMyMzM5Ng==","github-url":"https://github.com/perl11/cperl/issues/134#issuecomment-209323396"},"message":"With https://travis-ci.org/perl11/cperl/builds/122727369 linux passes, but darwin has a better allocator, and catches the invalid free.","files":null},{"type":5,"author":{"id":"04135dc995e2fdcd322aaf4268dbf04630a3cc7c"},"timestamp":1460960852,"metadata":{"github-id":"MDE0OlVubGFiZWxlZEV2ZW50NjMwNzk5MjQy"},"added":[],"removed":["blocking"]},{"type":3,"author":{"id":"04135dc995e2fdcd322aaf4268dbf04630a3cc7c"},"timestamp":1460960938,"metadata":{"github-id":"MDEyOklzc3VlQ29tbWVudDIxMTIyNDY3NQ==","github-url":"https://github.com/perl11/cperl/issues/134#issuecomment-211224675"},"message":"Removed blocking. It's only a PERL_DESTRUCT_LEVEL=2 issue in global destruction of that invalid pointer, so only observed by the core test suite, not in any real code, i.e. at scope end.","files":null}]}